client/src/components/JsonSchemaForm/fields/Description.tsx

```diff
@@ -18,19 +15,20 @@ Handlebars.registerHelper('abs', function(num) {
   return abs;
 });
 
-// compile once please!
-const compileTemplate = memoize(Handlebars.compile);
-
 function DescriptionField(props: Props) {
   const templateVars =  React.useContext(JsonSchemaFormTemplateContext);
-  const md = compileTemplate(props.description || "")(templateVars);
+  let description = props.description;
+
+  if (!description || typeof description !== 'string') {
+    description = '';
+  }
```

Comment:
Instead of this check, could you do

```
DescriptionField.defaultProps = {
   description: '',
};
```

Reply:
I tried it the other way, and typescript gets really unhappy.  I'd like to ship as is, and maybe you or @evinism can look at it afterwards to see if you can come up with a better solution.